docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md

```diff
@@ -142,12 +96,29 @@ You install Yugabyte Platform on a Kubernetes cluster as follows:
     helm install yw-test yugabytedb/yugaware --version 2.3.3 -n yb-platform --wait --set tls.sslProtocols="TLSv1.2"
     ```
 
+6. Check the services:
+
+    ```sh
+    kubectl get svc --namespace yb-platform
+    ```
+
 A message output will notify you whether or not the deployment is successful.
 
+# Add snippet to show the LB enable/disable
+
+
 ## Delete the Helm Installation of Yugabyte Platform
 
 To delete the Helm installation, run the following `helm del` command:
 
+**Helm v2.***
```

Comment:
we can remove the helm 2 section. If needed, let's mention helm 3 as a prerequisite in the beginning.